test/api/repositories/template-list/002-list-ok/expected-response.json

```diff
@@ -12,7 +12,10 @@
     {
       "id": "my-template-id",
       "name": "My Template",
-      "components": [],
+      "components": [
+        "foo.bar",
+        "keboola.wr-%s"
+      ],
```

Comment:
A API vrati konkretne `componentId` pre Snowflake Writer, pre dany stack.
Je to vzdy ta ista komponenta (kod) ale ma ine `componentId`, preto ze ma iny `staging-storage`.